### DISCUSSION TITLE
Adição da CMTV e atualização das Beach cams

### DISCUSSION THREAD

**Discussion by JpbmOfficial** [LITUATUI/M3UPT#78] [General]
Olá!
Seria possível adicionar o canal da CMTV e atualizar e adicionar novas beach cams? Há várias que não funcionam e muitas outras que ainda não estão adicionadas à lista.
Obrigado!

**Comment by LITUATUI** [comment]
Olá.

Não é possível adicionar o CMTV à lista pública pois o stream não é público. Quanto às beachcam se alguém quiser fazer um pull request com as actualizações eu aceito.
